db/migrate/20220614112810_add_devise_to_users.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+
+      # Uncomment below if timestamps were not included in your original model.
+      # t.timestamps null: false
+    end
+
+    add_index :users, :reset_password_token, unique: true
+    # add_index :users, :confirmation_token,   unique: true
+    # add_index :users, :unlock_token,         unique: true
+  end
+
+  def self.down
+    # By default, we don't want to make any assumption about how to roll back a migration when your
+    # model already existed. Please edit below which fields you would like to remove in this migration.
+    raise ActiveRecord::IrreversibleMigration
+  end
+end
```

Comment:
- [ ] Please check your migrations I am not able to run `rails db:migrate`
![migration](https://user-images.githubusercontent.com/44745855/180171033-672b3df9-b808-4e06-89a2-f55cb089d553.png)

Reply:
Hello @DeliceLydia 
I hope you are doing well. 
I don't really know which issue I am supposed to fix because everything is working fine on my side. I think you get this error because you are using my credentials while you should use yours on your computer.

I think the only thing you need to do is to:

1. Make sure the username and password exist in your local environment.(or create a new user and give him a password) 

2. Run the command:

``` Rails db:setup ```

Note: you are getting the error because you're trying to use the username and password of the unknown user.

Create a user first or use an existing user for your local environment